docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md

```diff
@@ -5,89 +5,212 @@ sidebar_label: Collect Logs
 description: Learn how to collect logs using the Sumo Logic OpenTelemetry Collector.
 ---
 
-The Sumo Logic Distribution for OpenTelemetry Collector provides various receivers for log collection. This document describes the receivers most commonly used for logs: the [Filelog Receiver](#filelog-receiver) and [Windows Log Event Receiver](#windows-log-event-receiver).
+The Sumo Logic Distribution for OpenTelemetry Collector provides various receivers for log collection. This document describes the receivers most commonly used for logs:
+
+
+* [Filelog Receiver](#filelog-receiver)
+* [Windows Log Event Receiver](#windows-log-event-receiver)
+* [Syslog Receiver](#syslog-receiver)
+
+You can find the full list of receivers on our [Sumologic OpenTelemetry Collector page](https://github.com/SumoLogic/sumologic-otel-collector/tree/main#components).
 
-You can find the full list of receivers on our [OpenTelemetry Collector GitHub page](https://github.com/SumoLogic/sumologic-otel-collector/tree/main#components).
 
 ## Filelog Receiver
 
-The Filelog Receiver tails and parses logs from files. The following is a basic configuration for the Filelog Receiver (collecting logs from a file), which you can place in the `conf.d` directory:
+Following configuration demonstrates : 
+ 1. Collect   : How to collect logs stored in /var/log/syslog and /var/log/audit/audit.log
+ 2. Transform : Set \_sourceCatetgory field to "application_logs_prod"
+ 3. Export    : Send data to authenticated Sumologic organization.
 
 ```yaml
 receivers:
-  filelog/custom_files:
-    include:
-    - /var/log/myservice/*.log
-    - /other/path/**/*.txt
-    - ./tmp/logs/*.log
-    include_file_name: false
-    include_file_path_resolved: true
+  filelog/custom_files:                                         
+    include_file_path: true
```

Comment:
Why changing this from `include_file_path_resolved: true` to `include_file_path: true`?
Why is the `include_file_name: false` being removed?